Dashboard/DashboardView.swift

```diff
@@ -0,0 +1,72 @@
+//
+//  Dashboard.swift
+//  AirCasting
+//
+//  Created by Lunar on 01/02/2021.
+//
+
+import SwiftUI
+import CoreData
+
+struct DashboardView: View {
+    
+    @State private var selectedView = SelectedSection.mobileActive
+    @Environment(\.managedObjectContext) var context
+    @StateObject private var coreDataHook = CoreDataHook()
+    
+    var sessions: [Session] {
+        coreDataHook.sessions
+    }
+    
+    var body: some View {
+        VStack {
+            
+            AirSectionPickerView(selection: $selectedView)
+
+            if sessions.isEmpty {
+                EmptyDashboardView()
+            } else {
+                ScrollView(.vertical) {
+                    LazyVStack(spacing: 20) {
+                        ForEach(sessions, id: \.uuid) { (session) in
+                            SessionCellView(session: session)
+                        }
+                    }
+                    .padding()
+                }
+                .frame(maxWidth: .infinity)
+                .background(Color.aircastingGray.opacity(0.05))
+            }
+        }
+        .navigationBarTitle("Dashboard")
+        .onChange(of: selectedView) { selectedSection in
+            
+            let request = NSFetchRequest<Session>(entityName: "Session")
```

Comment:
If we have a very nice abstraction that separates the core Data from the view layer maybe also move the predicate logic to `CoreDataHook` with